CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 ## 0.12.22
 
-* Introduce test `retry` option and corresponding configuration. Also
-  introduce related `no-retry` runner flag.
+* Introduce test `retry` option, corresponding `@Retry` annotation, and
+  corresponding `retry` test configuration to enable retries of failing tests.
```

Comment:
Please use full sentences in changelogs and documentation, and try to avoid assuming that the user knows the context of the package intimiately. For example,

> Add a `retry` option to `test()` and `group()` functions, as well as a `@Retry()` annotation for test files and a `retry` configuration field for `dart_test.yaml`. A test with retries enabled will be re-run if it fails for a reason other than a `TestFailure`.